public/locales/es/health.json

```diff
@@ -1,3 +1,3 @@
 {
-    "heading": "Página de Información de Salud"
+    "pageHeading": "Información de Salud"
```

Comment:
Also missing Spanish image link, so it's still the English version of the image.